v1.0/Specification.md

```diff
@@ -110,19 +110,19 @@ The following table describes the requirements for data quality, when preserving
 
 | ID | Name | Description |
 | --- | --- | --- |
-| ODS_1 | Password protection | The file **MUST NOT** have read or write password protection. |
-| ODS_2 | Standard compliance | The file **MUST** comply with the standard “ISO/IEC 29500 Office Open XML File Formats”. |
-| ODS_3 | Extension | The file **MUST** have extension “.fods”, “.ods” or “ots”. |
-| ODS_4 | Conformance | The file **SHOULD** comply with “strict” conformance class. |
-| ODS_5 | External data | The file **MUST NOT** have any references to external data. This includes data connections, RealTimeData functions, cell formula references and objects. |
-| ODS_6 | Embedded objects | The file **MUST NOT** have any embedded objects, which are in violation of your organisation’s file format policies. |
-| ODS_7 | Content | The file **SHOULD** have cell values or objects. |
-| ODS_8 | Macros | The file **SHOULD NOT** have any macros. |
-| ODS_9 | Absolute filepath | The file **SHOULD NOT** have absolute filepath to local file system in the XML attribute “x15ac:absPath” in “workbook.xml”. |
-| ODS_10 | Final | The file **SHOULD** have “final” mark. |
-| ODS_11 | Printer settings | The file **MAY** have XML attributes “PrinterName”, “PrinterPaperFromSetup” and “PrinterSetup”. |
-| ODS_12 | Metadata | The file **MAY** have metadata in the file’s properties or any user defined custom properties in “meta.xml”. |
-| ODS_13 | Cell hyperlinks | The file **MAY** have cell hyperlinks. |
+| OOXML_1 | Password protection | The file **MUST NOT** have read or write password protection. |
+| OOXML_2 | Standard compliance | The file **MUST** comply with the standard “ISO/IEC 29500 Office Open XML File Formats”. |
+| OOXML_3 | Extension | The file **MUST** have extension ".xlsm", “.xlsx”, “.xltx” or ".xltm". |
```

Comment:
Again, do you mean extension or file format here? A file can have any extension and you are talking about format in reality.